include/dap/network.h

```diff
@@ -17,6 +17,7 @@
 
 #include <functional>
 #include <memory>
+#include <stdint.h>
```

Comment:
Would `cstdint` have been more appropriate than `stdint.h` in this case? Some linters might complain about using the C header instead of the corresponding C++ header.

Reply:
Possibly. With caveats:

1. Convention-wise AFAIU `cppdap` uses only `stdint.h` form:

  ```
  $ git grep -F stdint.h | wc -l
  4
  $ git grep -F cstdint | wc -l
  0
  ```

  Those should probably better be switched all at the same time.

2. Standards-wise AFAIU `<cstdint>` only guaranteed to provide things in `std` namespace, like `std::uint32_t`. That's not what `cppdap` uses. `<stdint.h>` is expected to populate global namespace. (In practice both `gcc` and `clang` populate both namespaces in `<cstdint>`). To use those in a conformant way I think `cppdap` would need to switch to `std::` qualified identifiers.